chisel-wrapper

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+# USAGE:
+#   ./<script-name> [ chisel-cut-OPTIONS ] <slices..>
+# Installs the slices, using
+#   chisel cut "$@"
+# Generates a status file similar to /var/lib/dpkg/status at the CHISEL_DPKG_STATUS_FILE path
+# env vars:
+#   CHISEL_BIN                (optional) the path of chisel binary
+#   CHISEL_CACHE_DIR          (optional) the path which chisel will use to store it's cache
+#   CHISEL_DPKG_STATUS_FILE   (required) the path where the dpkg status file should be generated
+
+set -e
+
+CHISEL_BIN="${CHISEL_BIN:-"./chisel"}"
```

Comment:
shouldn't this be `CHISEL_BIN="${CHISEL_BIN:-"chisel"}"` instead? it feels awkward if I want to use the actual binary from PATH, to be doing `CHISEL_BIN=`which chisel` chisel-wrapper...`

Reply:
Yeah, it should. Thanks!